src/components/shared/index.js

```diff
@@ -85,6 +86,7 @@ const registerBootstrapComponents = (vue) => {
   vue.component('traction-card-header', BCardHeader)
   vue.component('traction-nav-item', BNavItem)
   vue.component('traction-nav', BNav)
+  vue.component('traction-input', BFormInput)
```

Comment:
This will still need to stay as TractionInput as this is being used across the board. What you can do is import BFormInput into the component you are using it in so it will isolate it.